File: web_server/web_server.js
var sys = require("sys"), http = require("http");
console.log("Starting server on port 8080")

http.createServer(function(request,response){
    response.writeHeader(200, {"Content-Type": "text/plain"});
    response.write("Request" + request.url);
    response.end();
}).listen(8080);

File: web_server/web_server.go
package main

import (
  "fmt"
  "net/http"
)

func handler(w http.ResponseWriter, r *http.Request) {
  fmt.Fprintf(w, "Request from %s", r.URL)
}

func main() {
  fmt.Println("Starting server on 8080")
  http.HandleFunc("/", handler)
  http.ListenAndServe(":8080", nil)
}
